src/vopr.zig

```diff
@@ -1101,7 +1107,6 @@ pub const Simulator = struct {
             assert(fault or header_prepare_view_mismatch);
         }
 
-        replica_storage.faulty = true;
```

Comment:
I think we can leave this as is? Seems __much__ more reabale to restore `faulty` in this function, no?

Reply:
https://github.com/tigerbeetle/tigerbeetle/pull/2334#discussion_r1773458812

Reply:
https://github.com/tigerbeetle/tigerbeetle/pull/2334/commits/7f7e1fda8341c4a494e13ee70d5f072d7e309919